src/unistd/src/lib.rs

```diff
@@ -90,24 +98,34 @@ pub extern "C" fn encrypt(block: [c_char; 64], edflag: c_int) {
     unimplemented!();
 }
 
-// #[no_mangle]
-// pub extern "C" fn execl(path: *const c_char, arg0: *const c_char /* TODO: , mut args: ... */) -> c_int {
-//     unimplemented!();
-// }
-//
-// #[no_mangle]
-// pub extern "C" fn execle(path: *const c_char, arg0: *const c_char /* TODO: , mut args: ... */) -> c_int {
-//     unimplemented!();
-// }
-//
-// #[no_mangle]
-// pub extern "C" fn execlp(file: *const c_char, arg0: *const c_char /* TODO: , mut args: ... */) -> c_int {
-//     unimplemented!();
-// }
+#[no_mangle]
+pub extern "C" fn _execl(path: *const c_char, args: *const *mut c_char) -> c_int {
```

Comment:
If you use `execv` like `musl`. You can remove this function. It has the same signature.